File: flutter_baidu_yingyan_trace-1.0.0/lib/src/model/entity/update_entity_result.dart
import 'package:flutter_baidu_yingyan_trace/src/model/base_result.dart';

/// 更新Entity响应结果
class UpdateEntityResult extends BaseResult {
  UpdateEntityResult(
      {required int tag, required int status, required String message})
      : super(tag: tag, status: status, message: message);

  /// map => UpdateEntityResult
  UpdateEntityResult.fromMap(Map map)
      : assert(map != null,
            'Construct a UpdateEntityResult，The parameter map cannot be null !'),
        super.fromMap(map);

  @override
  Map toMap() {
    return super.toMap();
  }

  @override
  String toString() {
    return "UpdateEntityResult{tag = $tag; status = $status; "
        "message = $message}";
  }
}

File: flutter_baidu_yingyan_trace-1.0.0/android/src/main/java/com/baidu/flutter/trace/model/entity/UpdateEntityResult.java
package com.baidu.flutter.trace.model.entity;

import com.baidu.flutter.trace.model.BaseResult;

/**
 * 更新Entity响应结果
 *
 * @author baidu
 */
public final class UpdateEntityResult extends BaseResult {

    /**
     * @param tag     响应标识
     * @param status  状态码
     * @param message 状态消息
     */
    public UpdateEntityResult(int tag, int status, String message) {
        super(tag, status, message);
    }

    @Override
    public String toString() {
        return "UpdateEntityResult [tag=" + tag + ", status=" + status + ", message=" + message
                + "]";
    }

}
